README.md

```diff
@@ -1,32 +1,32 @@
 # Maco - Malware config extractor framework
 
-Maco is a framework for **ma**lware **co**nfig extractors.
+## Maco is a framework for ***ma***lware ***co***nfig extractors.
 
-It aim for solve two problems:
-* Define a standardize ontology (or model) for extractor output. This greatly help for databasing extracted values.
-* Provide a standard way of identifying which parsers to run and how to execute it.
+It aims to solve two problems:
+* Define a standardize ontology (or model) for extractor output. This greatly helps for databasing extracted values.
+* Provide a standard way of identifying which parsers to run and how to execute them.
 
-Maco components
-* model.py
-    * a data model for common output of an extractor
-* extractor.py
+## Maco components
+* `model.py`
+    * a data model for the common output of an extractor
```

Comment:
The dot points above start with capitals. Did you want to standardise on this for the whole file?